src/commands/txCommands/util/gitHelpers.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require('chalk');
+const child_process = require('child_process');
+const Rx = require('rxjs');
+
+const child_process$ = Rx.Observable.bindNodeCallback(child_process.exec);
+
+/**
+ * Creates git commit of what has been staged
+ * @param  {String} resource resource slug to be used when we pull translations
+ * @param  {String} commitMessage what commit message to commit
+ * @return {Observable} Observable of tx process
+ */
+const commit$ = (resource, commitMessage) => {
```

Comment:
Since you're passing in the whole git command with `commitMessage`, there's actually nothing 'commit'-specific about `commit$` - I think it might be better to just pass in the actual commit message and hard-code `git commit -m ${JSON.stringify(commitMessage)}` in this function so that the caller can be reasonably certain that the code will be committed when the function is called?

If you need `--no-verify`, that could be passed in as an additional arg, and you could use the same signature pattern that [`child_process.spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) uses to have an optional array of args that can be passed to the child process

Reply:
ah yeah, mistake on my part in re-org yeah message should just be passed on entire command